### PR TITLE
Add several spelling corrections

### PR DESCRIPTION
**Pull request by luzpaz**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -12407,6 +12415,9 @@ frankin->franklin
 Fransiscan->Franciscan
 Fransiscans->Franciscans
 franzise->franchise
+frecuent->frequent
+frecuently->frequently
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:12419]: Can you also add a separate suggestion for the -ed version of this typo please.

Comment (peternewman) [reply codespell_lib/data/dictionary.txt:12419]: You could also add frequency (and -ies) versions of these too.

Comment (luzpaz) [reply codespell_lib/data/dictionary.txt]: like 5d9c9c4 ?

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -27894,6 +27910,7 @@ viwers->viewers
 vlarge->large
 vlaue->value
 vlaues->values
+vlone->clone
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:27913]: Can you add the clones and cloned versions of this typo too.

Comment (luzpaz) [reply codespell_lib/data/dictionary.txt:27913]: Done: 79af33c

**Pull request merged**